app/assets/javascripts/components/common/date_picker.jsx

```diff
@@ -9,10 +9,11 @@ const DatePicker = React.createClass({
 
   propTypes: {
     id: React.PropTypes.string,
-    value: React.PropTypes.string,
+    value: React.PropTypes.object, // should be a moment object
```

Comment:
That's weird. It works in the console on http://momentjs.com/docs/ — that is, `moment() instanceof moment` returns true — but it doesn't work in the dashboard.